framework/src/org/checkerframework/framework/util/typeinference/constraint/FIsA.java

```diff
@@ -21,7 +21,7 @@ public TUConstraint toTUConstraint() {
     @Override
     protected FIsA construct(
             AnnotatedTypeMirror newArgument, AnnotatedTypeMirror newFormalParameter) {
-        return new FIsA(newArgument, newFormalParameter);
+        return new FIsA(newFormalParameter, newArgument);
```

Comment:
For consistency and to avoid confusion, how about changing the order of arguments to either the `FIsA` constructor or to `AFConstraint.construct`?

Reply:
I was thinking about doing that, but that would have required more changes and potential confusion. This was the one location where error-prone complained and it seemed like a valid issue.